cmd/cert-checker/main.go

```diff
@@ -170,11 +177,12 @@ func (c *certChecker) checkCert(cert core.Certificate) (problems []string) {
 		if parsedCert.IsCA {
 			problems = append(problems, "Certificate can sign other certificates")
 		}
-		// Check the cert has the correct validity period +/- an hour
-		if parsedCert.NotAfter.Sub(cert.Issued) > checkPeriod+time.Hour {
-			problems = append(problems, "Certificate has a validity period longer than 90 days")
-		} else if parsedCert.NotAfter.Sub(cert.Issued) < checkPeriod-time.Hour {
-			problems = append(problems, "Certificate has a validity period shorter than 90 days")
+		// Check the cert has the correct validity period
+		expiryPeriod := parsedCert.NotAfter.Sub(parsedCert.NotBefore)
```

Comment:
I think this is where you fixed the bug, yeah?

I see you added a NotBefore: in the tests below, but it would be nice to ensure that the old behavior doesn't come back by making sure Issued and NotBefore are different or something and Issued would be invalid while NotBefore would be.

Or maybe something else, but involved Issued.